enhancements/upgrade-readiness.md

```diff
@@ -0,0 +1,580 @@
+---
+title: operator-status
+authors:
+  - "@awgreene"
+reviewers:
+  - "@ecordell"
+approvers:
+  - "@kevinrizza"
+creation-date: 2020-05-19
+last-updated: 2020-06-10
+status: implementable
+see-also:
+  - "N/A"  
+replaces:
+  - "N/A"
+superseded-by:
+  - "N/A"
+---
+
+# operator-status
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [x] Design details are appropriately documented from clear requirements
+- [x] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+
+## Summary
+
+The goal of this enhancement is to allow operators managed by the [Operator Lifecycle Manager (OLM)](https://github.com/operator-framework/operator-lifecycle-manager) operator to communicate conditions and states that OLM would otherwise be unable to infer. The presence of certain conditions should influence OLM behavior to protect the cluster.
+
+## Motivation
+
+OLM currently infers the state of an operator by inspecting the Kubernetes resources that define the operator. OLM does not provide operators with the means to communicate complex states that cannot be inferred without operator specific knowledge.
```

Comment:
nit: "OLM does not ... cannot ..." is double-negative.  Maybe simplify to:

> OLM does not currently provide operators with the means to communicate complex states.

or some such?